server/api-manager.js

```diff
@@ -926,11 +926,35 @@ class MindbodyQueries {
     let funcc = () => {
       return request.makeRequest();
     };
-    const backoff = (retries, fn, delay = 500) => fn().catch(err => (retries > 1 && !this.atLimit()) ? pause(delay).then(() => backoff(retries - 1, fn, delay * 2)) : Promise.reject(err));
+    function pause(milliseconds) {
```

Comment:
Madison's compact backoff function called this pause function, but it was undefined for me. It seems like Javascript, instead of crashing or sending a message, silently errored out when we wanted to delay, and so it never called backoff the 2nd, 3rd, 4th, 5th time. Maybe I'm wrong, though, since I seldom understand JS

Reply:
I saw that but thought the pause function was built into JavaScript and I thought it was pausing. It always takes a while to load, so I assumed it was pausing.